server/app/assets/javascripts/admin_programs.ts

```diff
@@ -103,16 +103,31 @@ class AdminPrograms {
       /* shouldDisable= */ disableNotificationPreferences,
     )
 
-    // Long program description
-    const longDescription = document.getElementById(
-      'program-display-description-textarea',
+    // External link
+    const externalLink = document.getElementById(
+      'program-external-link-input',
     ) as HTMLInputElement
+    const isNorthstarEnabled = externalLink.dataset.northstarEnabled === 'true'
+    const disableExternalLink =
+      (programType === ProgramType.DEFAULT ||
+        programType === ProgramType.COMMON_INTAKE_FORM) &&
+      isNorthstarEnabled
+    this.updateTextFieldElementDisabledState(
+      /* fieldElement= */ externalLink,
+      /* shouldDisable= */ disableExternalLink,
+    )
+    this.hideRequiredIndicators(
```

Comment:
suggestion (blocking): this is currently showing the required indicator for default and common intake programs when north star is disabled. this could be in it's own if check that just checks for `!ProgramType.EXTERNAL` since we never want the required indicator to show for other program types.